packages/embeds/embed-core/src/ModalBox/ModalBoxHtml.ts

```diff
@@ -74,10 +84,10 @@ ${getStyle()}
       <div id="skeleton" style="${skeletonStyle}" class="absolute z-highest">
         ${skeletonContent}
       </div>
-      <div id="error" class="hidden left-1/2 -translate-x-1/2 relative text-inverted"></div>
```

Comment:
Upgraded error to support any message instead. This was required by the new router API endpoint which when responds a message, we need to show that message